runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -330,4 +343,13 @@ public void onGameTick(GameTick event)
 			}
 		});
 	}
+
+
+	private void loadGELimits()
+	{
+		InputStream geLimitData = GrandExchangeSearchPanel.class.getResourceAsStream("/ge_limits.json");
+		itemGELimits = gson.fromJson(new InputStreamReader(geLimitData), new TypeToken<Map<Integer,Integer>>(){}.getType());
```

Comment:
TypeToken seems to be necessary. Occurrences exist in the project when dealing with data structures (like list) which have generics required